### PR TITLE
 Updates the BigQuery terraform script and documentation

### PR DESCRIPTION
**Pull request by martinjandl**
Jira: CT-XXX
Connection PR: XXX
SAPI PR: XXX

---

### CODE REVIEW

File: packages/php-db-import-export/.gitignore [hunk 1]
```diff
@@ -2,4 +2,11 @@
 /.idea
 /data
 composer.lock
-.env
\ No newline at end of file
+.env
+
+.phpunit.result.cache
+.terraform.lock.hcl
```
Comment (romanbracinik) [inline packages/php-db-import-export/.gitignore:8]: Lock by som commitoval https://github.com/keboola/storage-backend/pull/144/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R6-R9

**Review by romanbracinik — COMMENTED** [review]
jedna drobnost

File: packages/php-db-import-export/.gitignore (hunk 1 above)
Comment (martinjandl) [reply packages/php-db-import-export/.gitignore:8]: Jo aha. Upravim

**Review by romanbracinik — APPROVED** [review]
🙆‍♂️

**Pull request merged**